src/components/MaterialUI/muiThemes.test.js

```diff
@@ -0,0 +1,69 @@
+import createThemes from "./muiThemes";
+import { createMuiTheme } from "@material-ui/core/styles";
+
+describe("MUI Themes", () => {
+	it("createThemes returns expected themes", () => {
+		const applicationTheme = {
+			primary: {
+				main: "#1F5B7F",
+			},
+		};
+
+		const themeDefinition = {
+			direction: "ltr",
+			secondary: {
+				main: "#ED2E0B",
+			},
+			divider: "#CCCCCC",
+		};
+
+		const { applicationMuiTheme, muiTheme } = createThemes(
+			applicationTheme,
+			themeDefinition,
+		);
+
+		expect(applicationMuiTheme, "not to equal", null);
+		expect(muiTheme, "not to equal", null);
+
+		expect(applicationMuiTheme.primary.main, "to equal", "#1F5B7F");
+
+		expect(muiTheme.direction, "to equal", "ltr");
+		expect(muiTheme.secondary.main, "to equal", "#ED2E0B");
+		expect(muiTheme.divider, "to equal", "#CCCCCC");
+
+		expect(
+			muiTheme.overrides.MuiButton.outlined.border,
+			"to equal",
+			"1px solid rgba(0, 0, 0, 0.23)",
+		);
+	});
+
+	it("createThemes returns expected themes with dark palette type", () => {
+		const baseMuiTheme = createMuiTheme();
+
+		const themeDefinition = {
+			direction: "ltr",
+			secondary: {
+				main: "#ED2E0B",
+			},
+			palette: {
+				...baseMuiTheme.palette,
+				type: "dark",
+			},
+		};
+
+		const { applicationMuiTheme, muiTheme } = createThemes({}, themeDefinition);
+
+		expect(applicationMuiTheme, "not to equal", null);
+		expect(muiTheme, "not to equal", null);
+
+		expect(muiTheme.direction, "to equal", "ltr");
+		expect(muiTheme.secondary.main, "to equal", "#ED2E0B");
+
+		expect(
+			muiTheme.overrides.MuiButton.outlined.border,
+			"to equal",
+			"1px solid rgba(255, 255, 255, 0.23)",
+		);
+	});
```

Comment:
I don't think that it make sense to write unit tests on theme as I don't think it make sense to write tests on any styles at all. Argument is pretty simple - theme values cannot be an independent unit. I mean all those values exists only as part of context of other components/parts of application. As style cannot be an independent unit as well. Values for styles depends on context and correctness of those values is really relative.

Reply:
That brought test coverage at 100%, it does not hurt to have a test for that, does it?